useful-resources/privacy-policy.md

```diff
@@ -36,115 +24,192 @@ I dati personali raccolti rispettano il principio generale di "minimizzazione de
+medicina del lavoro, S.S. 17 est 12 F loc. S.Elia 67100 L'Aquila, P.IVA 01589150661 e codice fiscale 
+RBNSFN73H04F839L, ai fini di effettuare la visita medica per la concessione dell'idoneità allo 
+svolgimento della mansione lavorativa.
+
+| Nome                 | Ruolo                  | Scopo                                     |    
+|----------------------|------------------------|-------------------------------------------|    
+| Stefano Maria Rubino | Medico competente      | Visita e rilascio idoneità alla mansione  |
 
-In questa sezione descriviamo brevemente quali sono gli strumenti per la raccolta e la gestione dei dati personali, il loro utilizzo ed infine le persone che hanno accesso ai dati.
+## Responsabile per la sicurezza
 
-### FormKeep Form 
+Matteo Novelli è Il responsabile del servizio di prevenzione e protezione aziendale ed ha accesso ai 
+dati personali dei dipendenti ai fini della predisposizione di tutta la documentazione relativa alla 
+sicurezza nei luoghi di lavoro.
 
-[FormKeep's commitment to protect your data](https://intercom.help/formkeep/frequently-asked-questions/gdpr-compliance)
+| Nome             | Ruolo                  | Scopo                                                    |    
+|------------------|------------------------|----------------------------------------------------------|    
+| Matteo Novelli   | RSPP                   | Predisposizione di documenti relativi alla sicurezza sui luoghi di lavoro |
 
-- Form di contatto commerciale sito e pannello di gestione del consenso 
+### Dipendenti in Cloud
 
-Nome | Ruolo | Scopo |
----- | ----- | ----- |
-Matteo Latini | Co-founder | Gestione tecnica ed implementazione del form sul sito nebulab.it; gestione del consenso |
-Matteo Galliani | Partner | Gestione della fatturazione ; gesitone del consenso |
+[GDPR](https://www.dipendentincloud.it/gdpr/)    
 
-- Form di contatto candidature sito e pannello di gestione del consenso
+Software di gestione generica dei dipendenti, sul quali viene tenuto il registro presenze le 
+presenze, conservata la documentazione contrattuale e inserite mensilmente le buste paga.
 
-Nome | Ruolo | Scopo |
----- | ----- | ----- |
-Matteo Latini | Co-founder | Gestione tecnica ed implementazione del form sul sito nebulab.it; gestione del consenso |
-Matteo Galliani | Partner | Gestione della fatturazione ; gesitone del consenso |
+| Nome                  | Ruolo                | Scopo                   |    
+|-----------------------|----------------------|-------------------------|    
+| Andrea Cesare Iurisci | Resp. amministrativo | Gestione amministrativa |
+| Alberto Vena          | Co-founder           | Gestione amministrativa |
+| Davide Di Stefano     | Co-founder           | Gestione amministrativa |
+| Matteo Latini         | Co-founder           | Gestione amministrativa |
+| Matteo Galliani       | Partner              | Gestione amministrativa |
 
-### Slack
+Dati raccolti:
```

Comment:
Remove. We explain which data we collect in the `## Dati personali raccolti` section.